content/faculty/honorboard.md

```diff
@@ -0,0 +1,75 @@
+title: Доска почёта
+slug: honorboard
+category: faculty
+tags: [_nav, _category_header, faculty]
+submenu_index: 4
+---
+
+<ul class="nav nav-tabs" id="myTab">
```

Comment:
Dumb `id` attribute name. Anything similar to `honor-board-tab` would look better.